Code/MinimalLib/cffi_test.c

```diff
@@ -2164,6 +2164,51 @@ void test_partial_sanitization() {
   free(mpkl);
 }
 
+void test_capture_logs() {
+  printf("--------------------------\n");
+  printf("  test_capture_logs\n");
+  char *mpkl;
+  char *log_buffer;
+  void *null_handle = NULL;
+  size_t mpkl_size;
+  void *log_handle;
+  typedef struct {
+    const char *type;
+    void *(*func)(const char *);
+  } capture_test;
+  capture_test tests[] = {{"tee", set_log_tee}, {"capture", set_log_capture}};
```

Comment:
Is this legal C?

Reply:
Are you referring to the structure initialization? If so, yes, it is C99:
https://en.cppreference.com/w/c/language/struct_initialization